Sources/Plasma/PubUtilLib/plGLight/plShadowMaster.h

```diff
@@ -43,7 +43,8 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #ifndef plShadowMaster_inc
 #define plShadowMaster_inc
 
-#include "hsTemplates.h"
+#include <type_traits>
```

Comment:
Do we need this massive header?

Reply:
Whoops, I did originally but forgot to remove the include later